packages/python/plotly/plotly/tests/test_core/test_px/test_px_input.py

```diff
@@ -0,0 +1,29 @@
+import plotly.express as px
+import numpy as np
+import pandas as pd
+
+
```

Comment:
here I think we can also unit-test the `build_or_augment_dataframe` function directly... probably easier to read the tests that way IMO

Reply:
we can do both, it's also nice to have regression tests calling the px functions (which are here mostly to be sure it does not crash :-))